File: scripts/eoolo.php
#!/root/.phpenv/shims/php
<?php

function expr1()
{
  echo "method expr1\n";
  return "return expr1";
}

function expr2()
{
  echo "method expr2\n";
  return "return expr2";
}

var_dump(expr1() || expr2());

File: scripts/eoolo.pl
#!/root/.anyenv/envs/plenv/shims/perl

sub expr1 {
    print "method expr1\n";
    return "return expr1\n";
}

sub expr2 {
    print "method expr2\n";
    return "return expr2\n";
}

print &expr1 || &expr2
